javascript/ql/src/Security/CWE-078/UselessUseOfCat.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Useless use of cat
+ * @description Using cat to simply read a file can lead to unintended bugs, and at worst security issues.
+ * @kind problem
+ * @problem.severity error
+ * @precision high
+ * @id js/useless-use-of-cat
+ * @tags correctness
+ *       security
+ *       external/cwe/cwe-078
```

Comment:
I think we should drop the cwes here, and add `maintainability` instead. 

If we insist on cwes, then we should also have the ones for path-injection:

```
 *       external/cwe/cwe-022
 *       external/cwe/cwe-023
 *       external/cwe/cwe-036
 *       external/cwe/cwe-073
 *       external/cwe/cwe-099
```